src/main/java/se/iths/crimedatabase/entity/Crime.java

```diff
@@ -3,22 +3,46 @@
 import com.fasterxml.jackson.annotation.JsonFormat;
 import org.springframework.format.annotation.DateTimeFormat;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
 import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
 
 @Entity
 public class Crime {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
     private String name;
+
     @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm")
     @JsonFormat(pattern = "yyyy-MM-dd HH:mm")
     private LocalDateTime time;
 
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
If we remove a crime we also remove a category? (cascadetype)

Reply:
I didn't find any good resources... but you are correct in that we don't want to remove a category if we remove a crime that has that category. However, my thought was that the cascade typ "remove" (when used on a manyToOne relationships) only removes it if it doesn't have any connections left, i.e if we dont have any crimes with the category "Theft" than "Theft" gets removed from the category tabel. 

I'm not sure how it work, I think I will ask Martin if I don't find any good info.

Reply:
Martin said "good question" and than confessed that he doesn't know... 

After disscussing it a little we desided that it is best på remove thet cascade type "Remove" and let there be categories that doesn't have any crimes connected to them if all crimes that has a category are removed.